### DISCUSSION TITLE
The most frustrating experience ever

### DISCUSSION THREAD

**Discussion by stefax71** [community/community#138638] [GitHub Education]
### Hi everyone,

I believe the "Join GitHub Education" is one of the most frustrating experience I have ever had. It requires me to take a picture of a document and keeps rejecting my request even if the document is perfectly readable. 

I do not know about other online Universities, but UoPeople is online only and provides documents in the form of pdf files. I printed the confirmation of enrollment to take a picture of it, one of the most wicked things ever to be honest. It's readable, there is my name, the date and my student ID number. I used my **verified** University email address. Despite all of that, I keep being rejected. I wanted to use the transcript, but it's 6 pages of document, how can I take a picture of that? 

Does anyone have any advice to get the application through? I really tried 5 times now to no avail.

Thanks!

Stefano

**Comment by alx-mp4** [comment]
Hi @stefax71, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
